packages-ts/gauntlet-terra-contracts/src/lib/inspection.ts

```diff
@@ -12,3 +19,23 @@ export const getLatestOCRConfigEvent = async (provider: LCDClient, contract: Acc
 
   return setConfigTx?.logs?.[0].eventsByType['wasm-set_config']
 }
+
+export const getLatestOCRNewTransmissionEvent = async (provider: LCDClient, contract: AccAddress) => {
+  let transmissionTx = (
+    await provider.tx.search({
```

Comment:
I think this is complex enough to make a helper, like `getLatestContractEvent = (provider, event: string, contract: string)` ?